terraform/main.mk

```diff
@@ -7,7 +7,23 @@ ENV_DIR ?= $(INFRA_DIR)/env/$(ENV)
 OUTPUT_JSON_FILE = $(INFRA_DIR)/env/$(ENV)/output.json
 TERRAFORM_VERSION ?= "0.12.29"
 
-
+AWS_LIMITS_LIST ?= $$(echo "{ \
+\"aws_s3_bucket\":[ \
+	{\"value\":\"100\", \"service\":\"s3\", \"quotacode\":\"L-DC2B2D3D\"}], \
+\"aws_route53_health_check\":[ \
+	{\"value\":\"200\", \"service\":\"route53\", \"quotacode\":\"L-ACB674F3\"}], \
+\"aws_dynamodb_table\":[ \
+	{\"value\":\"256\", \"service\":\"dynamodb\", \"quotacode\":\"L-F98FE922\"}], \
+\"aws_eip\":[ \
+	{\"value\":\"5\", \"service\":\"vpc\", \"quotacode\":\"L-2AFB9258\"}] \
+}")
+
+AWS_LIMITS ?= @ ( echo $(foreach item, $(shell echo $(AWS_LIMITS_LIST) | $(JQ) -e -r '. | to_entries[] | .key' ), \
+$(shell echo $$(if [ $(shell grep -c "+ resource \"$(item)\"" $(ENV_DIR)/tfplan.txt) -lt $(shell echo $(AWS_LIMITS_LIST) | $(JQ) -r '.$(item)[].value') ]; \
+then echo "\n\033[32m[OK]\033[0m $(item) limit"; \
+else echo "\n\033[33m[WARNING]\033[0m $(item) limit \(Value:$(shell echo $(AWS_LIMITS_LIST) | $(JQ) -r '.$(item)[].value')\) exceeded! \
+Current value:$(shell grep -c "+ resource \"$(item)\"" $(ENV_DIR)/tfplan.txt) \
+\033[33m To request a service quota increase:\033[0m \033[36m aws service-quotas request-service-quota-increase --service-code $(shell echo $(AWS_LIMITS_LIST) | $(JQ) -r '.$(item)[].service') --quota-code $(shell echo $(AWS_LIMITS_LIST) | $(JQ) -r '.$(item)[].quotacode') --desired-value \<your_desired_value\> \033[0m"; fi ))) )
```

Comment:
Let's add another newline before the actual commands list

Reply:
@AutomationD you know yesterday I spent time to try to format output with new lines but due to the `echo` at the begining of `AWS_LIMITS` command it doesn't work, any ideas how to make output info with new lines?

Reply:
try `\n` ?

Reply:
@AutomationD I tried - no result, also tried to replace `echo` with `printf` - didn't work as well :(

Reply:
@AutomationD Output fixed, please approve
![image](https://user-images.githubusercontent.com/10009103/106011825-e154f100-60cb-11eb-98c4-96ff9018411e.png)